src/FileIO.scala

```diff
@@ -0,0 +1,55 @@
+import java.io.{File, PrintWriter}
+import java.nio.channels.Pipe.SourceChannel
+
+import scala.collection.mutable
+import scala.io.Source
+
+/**
+  * Created by joe on 10/18/16.
+  */
+object FileIO {
+
+  def prompt(s: String) = {
+    println(s);
+    io.StdIn.readLine()
+  }
+
+  def choice = {
+    val seq = Seq("Alcohol", "Furniture", "Shoes", "Toiletries", "Food", "Jewelry").mkString("\n")
+    prompt(s"\nPlease enter a category:\nAlcohol \nFurniture \nToiletries \nShoes \nFood \n Jewelry \n${seq}")
+
+  }
+
+  def findCategory(s: String) = {
+    purchases.filter(_._5 == s).map(x => {
+      println(s"Customer: ${x._1}, Date: ${x._2.substring(0, 10)}")
+      val writeToFile = new PrintWriter(new File(s"${s}_filtered_purchases.prn"))
```

Comment:
need to rewrite this file output. Think about how it will run step by step and you should realize why it needs to be rewritten.  :)